src/index.html

```diff
@@ -1,13 +1,338 @@
+                Why buy direct from Bose
+              </h2>
+            </div>
+            <div class="grid__item grid__item--6-10 grid__item--tablet--4-6">
+              <h3 class="page__section-subtitle">Free 2-day shipping and returns</h3>
+              <h3 class="page__section-subtitle">90-day risk-free trial</h3>
+              <h3 class="page__section-subtitle">World class customer service</h3>
+              <h3 class="page__section-subtitle">My Bose account management</h3>
+              <p>
+                A great product is more than what’s in 
+                the box. It’s a promise of premium performance,
+                world-class support, and everything you expect
+                from a trusted brand. It’s just one of many
+                reasons why you’ll shop with confidence on Bose.com.
+              </p>
+            </div>
+          </div>
+        </div>
+      </section>
+      <section class="page__section contact-us" id="contacts">
+        <div class="container">
+          <h2 class="page__section-title">
+            Contact us
+          </h2>
+          <div class="grid grid--mobile">
+            <div class="grid__item grid__item--1-5 grid__item--tablet--1-3">
+              <form action="#" method="get">
+                <input
+                  name="name"
+                  type="text"
+                  placeholder="Name"
+                  class="contact-us__form-field"
+                  required
+                >
+                <input
+                  name="email"
+                  type="email"
+                  placeholder="Email"
+                  class="contact-us__form-field"
+                  required
+                >
+                <textarea
+                  name="message"
+                  placeholder="Message"
+                  minlength="1"
+                  class="contact-us__form-field contact-us__form-field--textarea"
+                  required
+                ></textarea>
+
+                <button class="button contact-us__button">
+                  Send
+                </button>
+              </form>
+            </div>
+            <div class="grid__item grid__item--8-10 grid__item--tablet--4-6">
+              <div class="contact-us__info">
+                <div class="contact-us__title">
+                  Phone
+                </div>
+                <a
+                  href="tel:+1 234 5555-55-55"
+                  class="contact-us__link"
+                >
+                  +1 234 5555-55-55
+                </a>
+                <div class="contact-us__title">
+                  Email
+                </div>
+                <a
+                  href="mailto:hello@miami.com"
+                  class="contact-us__link"
+                >
+                  hello@miami.com
+                </a>
+                <div class="contact-us__title">
+                  Address
+                </div>
+                <a
```

Comment:
Cheklist | point 9 => Location-related addresses / links should open google maps in a new tab target="_blank"